container/container_unit_test.go

```diff
@@ -66,3 +69,59 @@ func TestContainerSecretReferenceDestTarget(t *testing.T) {
 		t.Fatalf("expected secret dest %q; received %q", expected, d)
 	}
 }
+
+func TestContainerLogPathSetForFileLoggers(t *testing.T) {
```

Comment:
As much as I'd prefer a unit test, I think this one is more appropriate as an integration test.
@dnephin WDYT?

Reply:
I think a unit test is appropriate here. I really think we should only be writing integration tests for features, not bug fixes.

Reply:
Works for me.